<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogOptions</class>
 <widget class="QDialog" name="DialogOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Programs (Context Menu)</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="minimumSize">
          <size>
           <width>850</width>
           <height>0</height>
          </size>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
         </property>
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>175</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <row>
          <property name="text">
           <string>Neue Zeile</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>edit</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>File Type</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Titel (Menu)</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Program</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Arguments</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Directory Mode</string>
          </property>
         </column>
         <item row="0" column="1">
          <property name="text">
           <string>Editor</string>
          </property>
         </item>
         <item row="0" column="2">
          <property name="text">
           <string>/usr/bin/kate</string>
          </property>
         </item>
         <item row="0" column="3">
          <property name="text">
           <string>${full}</string>
          </property>
         </item>
         <item row="0" column="4">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string>Shell</string>
          </property>
         </item>
         <item row="1" column="2">
          <property name="text">
           <string>/usr/bin/bash</string>
          </property>
         </item>
         <item row="1" column="3">
          <property name="text">
           <string/>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonAdd">
           <property name="text">
            <string>&amp;Add/change</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonDel">
           <property name="text">
            <string>D&amp;elete</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonUp">
           <property name="text">
            <string>&amp;Up</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonDown">
           <property name="text">
            <string>&amp;Down</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Current Entry</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>File Type:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Title:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Program:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Arguments:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Directory mode</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QComboBox" name="comboBoxFileType">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>File</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Directory</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Both</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditTitle">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QComboBox" name="comboBoxProgram">
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonSelectProgram">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../reditor/editor.qrc">
                <normaloff>:/main/icons/layout_add.png</normaloff>:/main/icons/layout_add.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="3">
           <widget class="QComboBox" name="comboBoxArgument">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string notr="true">${full}</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">${path}</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">${node}</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">${ext}</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QComboBox" name="comboBoxDirMode">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Parent directory</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Entry (if directory)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Base directory</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogOptions</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogOptions</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
